### PR TITLE
FSM legend

### PR DESCRIPTION
**Pull request by chowington**
Fixes https://github.com/VEuPathDB/web-eda/issues/1347
Requires https://github.com/VEuPathDB/web-components/pull/394

### CODE REVIEW

File: src/lib/core/components/fullScreenApps/fullScreenMap.tsx [hunk 1]
```diff
@@ -169,7 +173,9 @@ function FullScreenMap(props: FullScreenComponentProps) {
     xAxisVariable: appState.overlayVariable,
     computationType: 'pass',
     markerType: appState.markerType,
-    //TO DO: checkedLegendItems and maybe dependentAxisLogScale
+    // Endpoint can't currently handle checkedLegendItems
+    // checkedLegendItems: appState.checkedLegendItems,
```
Comment (dmfalke) [inline src/lib/core/components/fullScreenApps/fullScreenMap.tsx:177]: I tested uncommenting this line, and things seem to work. Can you elaborate on what isn't working? Thanks!

**Review by dmfalke — COMMENTED** [review]
I've tested this, and it works great! I asked a question about handling unchecking legend items.

File: src/lib/core/components/fullScreenApps/fullScreenMap.tsx (hunk 1 above)
Comment (moontrip) [reply src/lib/core/components/fullScreenApps/fullScreenMap.tsx:177]: @dmfalke I have tested this by uncommenting the line, and it seems to work fine for me too.

Comment (chowington) [reply src/lib/core/components/fullScreenApps/fullScreenMap.tsx:177]: Oh, when I tested it, I couldn't get any overlay variable info or even counts on the markers. If it works for y'all though, that's great!

**Review by dmfalke — APPROVED** [review]
Great! Let's go ahead and merge!

**Comment by moontrip** [comment]
> Great! Let's go ahead and merge!
> 
> ETA after uncommenting the checked legend items being passed to useMapMarkers.

@dmfalke I think you can do that as Connor is on vacation this week - or I can do that on his behalf. Just let me know if you prefer the latter 😃

**Comment by moontrip** [comment]
Oh by the way this PR at web-conponents need to be merged at first though. https://github.com/VEuPathDB/web-components/pull/394

**Comment by moontrip** [comment]
@dmfalke well I can do that for both this PR and web-components one 😃 Will do that now

**Comment by dmfalke** [comment]
Oh, I forgot he was on vacation! Feel free to merge. I can get to it later, if you don't.

**Comment by moontrip** [comment]
@dmfalke haha no problem I'm working on it

**Pull request merged**